### ISSUE TITLE
Document the correct way to callibrate the compass

### ISSUE THREAD

**Issue by ntoll** [bbcmicrobit/micropython#59]
Apparently, the correct way to calibrate the compass is to turn it flat on the spot (without tilting it otherwise that will add inaccuracies). We should document this in an obvious place.

**Comment by willingc** [comment]
@ntoll Perhaps the easiest way to document initially is in an introductory docstring block:

```
"""
    compass.py
    ~~~~~~~~~

    Creates a compass. To calibrate the compass, lie it flat and spin it horizontally. For 
    best results during calibration, try not to tilt the compass up and down.

"""
```

**Comment by ntoll** [comment]
+1 - also there should be something in the api so `help(microbit.compass.callibrate)` returns something useful that an 11yo kid would understand.

**Comment by ntoll** [comment]
Docstring added, the `help` related text still doesn't reference how to calibrate it. This will be fixed by #125.
